### PR TITLE
Improves test coverage

### PR DESCRIPTION
**Pull request by skw**
closes github/memex#8949

## Description

This adds tests to more completely cover `addToProject` test cases, including:
- handling multiple label filters
- throwing errors when invalid project-url inputs are set
- ownerTypes are correctly set on graphQL queries

### CODE REVIEW

File: __tests__/add-to-project.test.ts [hunk 1]
```diff
@@ -179,6 +179,209 @@ describe('addToProject', () => {
...
+    expect(gqlMock).toHaveBeenCalled()
+    expect(infoSpy).not.toHaveBeenCalled()
+    expect(outputs.itemId).toEqual('project-next-item-id')
+  })
+
+  test('does not add un-matching issues with multiple label filters', async () => {
+    mockGetInput({
+      'project-url': 'https://github.com/orgs/github/projects/1',
+      'github-token': 'gh_token',
+      labeled: 'accessibility,backend,bug'
```
Comment (mattcosta7) [inline __tests__/add-to-project.test.ts:233]: what happens if the label has a `,` in it? can we support that?

do we need to consider other syntax to support that?  What about other characters, spaces, etc?

**Comment by mattcosta7** [comment]
This is looking really good so far!

File: __tests__/add-to-project.test.ts (hunk 1 above)
Comment (skw) [reply __tests__/add-to-project.test.ts:233]: Luckily labels can't be created with `,` but good point nonetheless about spaces and other edge cases 👍

Comment (camchenry) [reply __tests__/add-to-project.test.ts:233]: Bit off-topic to this PR, but would be great to pass off all of these concerns to a separate library just for filtering (e.g., https://github.com/github/memex-filter-parser) and have a single `filter:` input here.

Comment (skw) [reply __tests__/add-to-project.test.ts:233]: @camchenry I'm getting a `404`

Comment (mattcosta7) [reply __tests__/add-to-project.test.ts]: @camchenry eventually that might make sense, but for this specific implementation, we only support a handful of fields, `label` `assignee` `milestone` (and I think that's probably it)

extracting the parser could be fun though

**Review by mattcosta7 — APPROVED** [review]
